packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -83,7 +83,7 @@ const ToggleMenuDrawer = observer(({ platform_config }) => {
                 ];
             } else if (location === routes.traders_hub || is_trading_hub_category) {
                 primary_routes = [routes.account, routes.cashier];
-            } else if (location === routes.wallets) {
+            } else if (location === routes.wallets || is_next_wallet_enabled) {
```

Comment:
is it possible to be on routes.wallets when is_next_wallet_enabled is false? feels redundant unless I do not get the logic fully